src/infrastructure/logging.py

```
@@ -0,0 +1,153 @@
+import logging
+import logging.handlers
+import json
+import sys
+from pathlib import Path
+from datetime import datetime
+from typing import Dict, Any, Optional
+import traceback
+from pythonjsonlogger import jsonlogger
```

Pylint (3.3.7)

[error] Unable to import 'pythonjsonlogger'

(E0401)